lib/streamlit/components/v1/__init__.py

```diff
@@ -12,15 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# `html` and `iframe` are part of Custom Components, so they appear in this
-# `streamlit.components.v1` namespace.
 import streamlit
-from streamlit.components.v1.components import declare_component
 
+# The `custom_component as components` import exists as existing custom components have started
+# to rely on internals of the components package. For example, streamlit-option-menu accesses
+# [register_widget](https://github.com/victoryhb/streamlit-option-menu/blob/master/streamlit_option_menu/streamlit_callback.py#L28),
```

Comment:
uff, thats not good. `streamlit_option_menu` has quite a high usage. We probably need to investigate this a bit further to see what we need to provide to get this patch removed from `streamlit_option_menu`.

Reply:
yep,  I already talked with @sfc-gh-jcarroll and we agreed that we want to look more into this in context of the broader custom components work.